site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -2635,7 +2645,19 @@ public int compare(JoinableGroup g1, JoinableGroup g2){
 
 			context.put("authAllowed", ServerConfigurationService.getBoolean("sitemanage.grant.auth", false));
 			context.put("anonAllowed", ServerConfigurationService.getBoolean("sitemanage.grant.anon", false));
-
+			SimpleDateFormat iso8601 = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+			iso8601.setTimeZone(userTimeService.getLocalTimeZone());
+			SimpleDateFormat readableDate = new SimpleDateFormat("MM/dd/yyyy hh:mm aa");
+			readableDate.setTimeZone(userTimeService.getLocalTimeZone());
```

Comment:
we need UserTimeService here as well

Please let me know if you want me to implement it in your branch or via a PR